L1Trigger/L1TMuonEndCap/src/MicroGMTConverter.cc

```diff
@@ -24,6 +25,10 @@ void MicroGMTConverter::convert(
   out_cand.setHwQual          ( in_track.GMT_quality() );
   out_cand.setHwHF            ( false );  // EMTF: halo -> 1
   out_cand.setTFIdentifiers   ( sector, tftype );
+  // Truncate to 11 bits and offset by 25 from global event BX in EMTF firmware
```

Comment:
This code does not belong here. Instead it should replace line 85. https://github.com/cms-sw/cmssw/pull/22090/files#diff-0098191652eebd75579da4dd04b68427R85

Reply:
I would not put the offset in "convert_all", because "convert" can be called stand-alone, and I would not want to have to perform the truncation and shift every time I call "convert".

Reply:
Line 85 is not in convert_all but in this function.
`out_cand.setTrackSubAddress(l1t::RegionalMuonCand::kBX , in_track.BX());` is written there at the moment, and this should be replaced because it sets the kBX back to the old value.